ledger-core/network/nds/uniproto/l2/uniserver/dispatcher.go

```diff
@@ -34,6 +34,7 @@ type Dispatcher struct {
 	started     uniproto.ProtocolSet
 }
 
+// todo: ??? unused
```

Comment:
the comment is irrelevant as this whole PR is unused by now
should revisit only after a few months of use